packages/components/pie-link/test/component/pie-link.spec.ts

```diff
@@ -1,58 +1,61 @@
-import { test, expect } from '@sand4rt/experimental-ct-web';
-import { PieLink, type LinkProps } from '../../src/index.ts';
-import { tags } from '../../src/defs.ts';
+import { test, expect } from '@playwright/test';
+import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+import { type LinkProps, tags } from '../../src/defs.ts';
 
-const componentSelector = '[data-test-id="pie-link"]';
+import { link } from '../helpers/page-object/selectors.ts';
 
-const anchorProps: LinkProps = {
-    tag: 'a',
-    href: '#',
-    target: '_blank',
-    rel: 'nofollow',
-};
+const componentSelector = '[data-test-id="pie-link"]';
 
 test.describe('PieLink - Component tests', () => {
-    test('should be visible', async ({ mount, page }) => {
+    test('should be visible', async ({ page }) => {
         // Arrange
-        await mount(PieLink, {
-            slots: { default: 'Link!' },
-        });
+        const linkPage = new BasePage(page, 'link');
+        const props: LinkProps & { slot: string } = {
+            slot: 'Link',
+        };
+
+        await linkPage.load({ ...props });
```

Comment:
You shouldn't have to pass this slot as the story has one set by default

```suggestion
        await linkPage.load();
```